examples/ecommerce-jewellery-store/src/components/ExpiryDate.tsx

```diff
@@ -0,0 +1,9 @@
+import { DateInput } from "@progress/kendo-react-dateinputs";
+
+const ExpiryDate = () => {
```

Comment:
I generally agree, however, in vNext we can introduce some custom logic to make it more real-world-like. In that regard, I believe it will be more beneficial to still have this component, where, in the future, we can add our logic.